web_widget_x2many_2d_matrix/static/src/js/web_widget_x2many_2d_matrix.js

```diff
@@ -50,10 +53,18 @@ openerp.web_widget_x2many_2d_matrix = function(instance)
             this.field_y_axis = node.attrs.field_y_axis || this.field_y_axis;
             this.field_label_x_axis = node.attrs.field_label_x_axis || this.field_x_axis;
             this.field_label_y_axis = node.attrs.field_label_y_axis || this.field_y_axis;
+            this.x_axis_clickable = node.attrs.x_axis_clickable != undefined ? this.parse_boolean(node.attrs.x_axis_clickable) : this.x_axis_clickable;
+            this.y_axis_clickable = node.attrs.y_axis_clickable != undefined ? this.parse_boolean(node.attrs.y_axis_clickable) : this.y_axis_clickable;
```

Comment:
I'd be cautious about 'correctly', there are a lot of weirdnesses like empty arrays not being falsy and similar things. But for checking if something is set `value || default` works perfectly fine, and then you don't need to go into the details of `undefined`, `null`, `false` etc, as all of those are falsy